.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,63 @@
+name: publish-to-pypi
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
would be better to make it run on a regex X.Y.Z (X,Y,Z integers)

Reply:
Changed